gnomad_hail/utils/annotations.py

```diff
@@ -419,3 +419,63 @@ def annotate_freq(
 
     # Return MT with freq row annotation
     return mt.annotate_rows(freq=freq_expr).drop('_freq_meta')
+
+
+def get_annotations_hists(
+    ht: hl.Table, annotations_hists: Dict[str, Tuple],
+    log10_annotations: List[str] = ['DP']
+    ) -> Dict[str, hl.expr.StructExpression]:
+    """
+    Creates histograms for variant metrics.
+    Used when creating site quality distribution json files.
+
+    :param Table ht: Table with variant metrics
+    :param Dict annotations_hists: Dictionary of metrics names and their histogram values (start, end, bins)
+    :param List log10_annotations: List of metrics to log scale
+    :return: Dictionary of metrics and their histograms
+    :rtype: Dict[str, hl.expr.StructExpression]
+    """
+    hist_dict = {}
+
+    # Check all fields in ht.info and create histograms if they are in annotations_hists dict
+    info_fields = list(ht.row.info)
```

Comment:
unnecessary: you can directly write `for field in ht.row.info:`

Reply:
Also -and up to you if you like/dislike it- but you could write this like that if you want:
```
return {
    field: hl.agg.hist(
            hl.log10(ht[field] if field in log10_annotations else ht[field), start, end, bin
    ) for field, (start, end, bins) in annotations_hists.items() if field in ht.row.info
}
```

Reply:
ooh, nice!